tests/debug.test.js

```diff
@@ -50,4 +52,24 @@ describe('debugCommand()', () => {
         sinon.assert.calledWithMatch(fakeProcess.stdout.write, 'msgstr[1] "{0} :sƃɐʇ pıʅɐʌuI');
       });
   });
+
+  it('should support keepOriginalHeader', () => {
+    const writeFileMock = sinon.spy();
+
+    const poToolsModule = rewire('./node_modules/pofile/lib/po.js');
+
+    const revertMock = poToolsModule.__set__('fs', {
+      writeFile: writeFileMock,
+    });
+
+    return debugCommand({potFile: './tests/fixtures/source.pot', format: 'unicode', output: './tests/fixtures/output.po'}, {_chalk: fakeChalk, _process: fakeProcess, _console: fakeConsole})
```

Comment:
> I would avoid using rewire, it's not necessary. I'd use the same convention as these other tests and look at what's being written to the process stub. That will get your test up and running.

I'm afraid that won't be possible here. The way this works now is that we are copying the header from the `output` .po file, so I need to set that which means that we don't call `_process.stdout.write` but `po.save()` instead. That's why I wanted to mock this.